equinix/resource_fabric_connection.go

```diff
@@ -0,0 +1,265 @@
+package equinix
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"strings"
+	"time"
+
+	v4 "github.com/equinix-labs/fabric-go/fabric/v4"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+)
+
+func resourceFabricConnection() *schema.Resource {
+	return &schema.Resource{
+		Timeouts: &schema.ResourceTimeout{
+			Create: schema.DefaultTimeout(6 * time.Minute),
+			Update: schema.DefaultTimeout(10 * time.Minute),
+			Delete: schema.DefaultTimeout(6 * time.Minute),
+			Read:   schema.DefaultTimeout(6 * time.Minute),
+		},
+		ReadContext:   resourceFabricConnectionRead,
+		CreateContext: resourceFabricConnectionCreate,
+		UpdateContext: resourceFabricConnectionUpdate,
+		DeleteContext: resourceFabricConnectionDelete,
+		Importer: &schema.ResourceImporter{
+			StateContext: schema.ImportStatePassthroughContext,
+		},
+		Schema: createFabricConnectionResourceSchema(),
+
+		Description: "Resource allows creation and management of Equinix Fabric	layer 2 connections",
```

Comment:
It would be good for users to specify in the description this resource is for Fabric v4. Same for all other `fabric_` resources/datasources